### ISSUE TITLE
Joomla For All People - Badge proposal

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/accessibility#33]
## Introduction
We want to encourage and mobilize designers to make Joomla extensions and templates as accessible as it can be.
We also want to make sure that Joomla users know whether the extensions or templates meets accessibility requirements.

Our goal is to have the Joomla 4 and extensions or templates for Joomla4 compliant with accessibility requirements, ready to build accessible sites.

We are considering a proposal for designers to declare compatibility of WCAG 2.0 extensions placed in Joomla Extension Directory. 

Drupal has its own [#D8AX hashtag (Drupal 8 Accessibility eXperience)](https://groups.drupal.org/node/66323). Wordpress has a tag [Accessibility Ready](https://make.wordpress.org/themes/handbook/review/accessibility/) for their themes.

**I propose the creation of a badge "Joomla for all ".**

## Rules:
1. The developer can mark his extension or template with a J4A badge.
2. The developer includes the following statement on the project page, including Joomla Extension Directory or the planned Joomla Template Directory:
> We have tried to make this extension/template *as accessible for all* as possible. If you find any defects, *let us know*. We will fix them.

where

* the words *as accessible for all* as possible are a link between the "Joomla Accessibility Code" (or Accessibility Statement).
* the words *let us know* are a link to the contact form with developer or him issue tracker.

3. The developer may extend the declaration with a list of accessibility features and evaluation tools.
4. The JAT will carry out regular audits of the accessibility of extensions and templates and publish reports in the Joomla! Community Magazine.

## What do we have to do?
1. Developing the Joomla Accessibility Code
2. Developing badge
3. Popularize the badge

**Please discuss!**

**Comment by coolcat-creations** [comment]
We have the conditional use logos, so the badge should match their style. I sent you the file in glip.

**Comment by zwiastunsw** [comment]
Proposal for a shorter badge name: Joomla! For All

**Comment by hvdmeer** [comment]
Will the word Accessible but used in the wording? Because Joomla! for all (people) can mean just about anything.

**Comment by chmst** [comment]
I agree, if the context is not given, "for all" could be everything.  I use to say "Accessible for all". It is good to pronounce, kind of music in it ;)

**Comment by zwiastunsw** [comment]
Thanks for your comments. Please note that there is no place for the novel on the badge :) . Information is not only an inscription but also icons. In addition, the badge will be accompanied by a declaration of accessibility.

**Comment by brianteeman** [comment]
dont know about the wordpress one but the drupal one isnt used.
